src/app/resources/curriculum/curriculum.component.html

```diff
@@ -5,8 +5,8 @@ <h3>Slides &amp; Worksheets</h3>
       <tr *ngFor="let topic of topics">
         <td style="text-align: left;">{{topic.name}}</td>
         <ng-template [ngIf]="!topic.areLinksEnabled" [ngIfElse]="available">
-          <td><font style="color: #999">slides</font></td>
-          <td><font style="color: #999">worksheet</font></td>
+          <td><span class='slidesAndWorksheets'>slides</span></td>
+          <td><span class='slidesAndWorksheets'>worksheet</span></td>
```

Comment:
So much cleaner! 🎉 

Looking at line 13 it looks like we usually wrap class names in double quotes in html files. Could we update that for consistency?